docs/conf/australia/2019/news/cfp-reminder.rst

```diff
@@ -0,0 +1,42 @@
+:template: {{year}}/generic.html
+
+.. post:: August 7, 2019
+   :tags: australia-2019, website, cfp, tickets
+
+
+CFP Reminder & Tickets on Sale
+==============================
+
+G'day fellow documentarians!
+
+Here is the latest update about our Australian conference.
+Our `call for proposals <https://www.writethedocs.org/conf/australia/2019/cfp/>`_ ends at midnight August 9, that is *tomorrow*!
```

Comment:
It will be *today* once the email goes out :/ Let's ship it today tho, and people will get it when they wake up.

Reply:
How do you feel about 

> which is like, right now! :-D 

?

Reply:
Seems fine.